Ginger/GingerTest/POMs/JavaPOM.cs

```diff
@@ -0,0 +1,404 @@
+﻿using amdocs.ginger.GingerCoreNET;
+using Amdocs.Ginger.Common;
+using Amdocs.Ginger.Common.UIElement;
+using Amdocs.Ginger.Repository;
+using Ginger.Run;
+using GingerCore;
+using GingerCore.Actions;
+using GingerCore.Actions.Common;
+using GingerCore.Drivers.CommunicationProtocol;
+using GingerCore.Drivers.JavaDriverLib;
+using GingerCore.Platforms;
+using GingerCoreNET.SolutionRepositoryLib.RepositoryObjectsLib.PlatformsLib;
+using GingerTestHelper;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Media.Imaging;
+
+namespace GingerTest.POMs
+{
+    [TestClass]
+    [Level3]
+    public class JavaPOMsTest
+    {
+        static GingerWPF.WorkSpaceLib.WorkSpaceEventHandler WSEH = new GingerWPF.WorkSpaceLib.WorkSpaceEventHandler();
+
+        static POMsPOM mPOMsPOM = null;
+        static ApplicationPOMModel mLearnedPOM = null;
+        static List<ElementLocator> prioritizedLocatorsList = null;
+
+        static BusinessFlow mBF;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the field 'mBF' and declare it as a local variable in the relevant methods.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=4423505)